src/RootNavigator.tsx

```diff
@@ -61,14 +53,7 @@ export default React.memo(function RootNavigator() {
     <>
       <Stack.Navigator
         screenOptions={{
-          headerStyle: {
```

Comment:
All of these are gone. Why?

Reply:
Ah, probably because we now use the RN-paper widget instead?

Reply:
Yes, the theme is already applied and I didn't implement features we don't need, so we don't need to deactivate them anymore.